recipes/pyutilib/meta.yaml

```diff
@@ -87,8 +87,6 @@ test:
     # entry points work.
 
     - test.pyutilib --help
-    - lbin --help
-    - lpython --help
```

Comment:
What I normally do then is use `command -v` on Mac and Linux and `where` on Windows. This should verify that they do exist even if we aren't exercising them so much.